testdata/p4_16_samples_outputs/table-entries-exact-ternary-bmv2.p4.entries.txt

```diff
@@ -0,0 +1,124 @@
+updates {
+  type: INSERT
+  entity {
+    table_entry {
+      table_id: 33616996
+      match {
+        field_id: 1
+        exact {
+          value: "\001"
+        }
+      }
+      match {
+        field_id: 2
+        ternary {
+          value: "\021\021"
+          mask: "\000\017"
```

Comment:
Does BMv2 simple_switch and/or P4Runtime allow values in value/masks where (value & mask) != value ?  Should the compiler be warning/erroring about this?